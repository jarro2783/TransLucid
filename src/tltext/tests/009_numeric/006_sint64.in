%%
sint64"0";;
sint64"1";;
sint64"12";;
sint64"9223372036854775807";;
sint64"-9223372036854775808";;

sint64"0" + sint64"0";;
sint64"15" + sint64"0";;
sint64"0" + sint64"15";;
sint64"8" + sint64"5";;
sint64"-128" + sint64"130";;
sint64"-128" + sint64"5";;

sint64"0" - sint64"0";;
sint64"10" - sint64"0";;
sint64"0" - sint64"10";;
sint64"15" - sint64"3";;

sint64"6" * sint64"1";;
sint64"6" * sint64"0";;
sint64"-5" * sint64"4";;
sint64"-5" * sint64"-4";;

sint64"5" / sint64"1";;
sint64"23" / sint64"5";;
sint64"-10" / sint64"2";;

sint64"7" % sint64"4";;

sint64"0" < sint64"1";;
sint64"1" < sint64"1";;
sint64"2" < sint64"1";;

sint64"0" <= sint64"1";;
sint64"1" <= sint64"1";;
sint64"2" <= sint64"1";;

sint64"0" > sint64"1";;
sint64"1" > sint64"1";;
sint64"2" > sint64"1";;

sint64"0" >= sint64"1";;
sint64"1" >= sint64"1";;
sint64"2" >= sint64"1";;

sint64"0" == sint64"1";;
sint64"1" == sint64"1";;
sint64"2" == sint64"1";;

sint64"0" != sint64"1";;
sint64"1" != sint64"1";;
sint64"2" != sint64"1";;

sint64"5"!prec;;
sint64"5"!is_signed;;
