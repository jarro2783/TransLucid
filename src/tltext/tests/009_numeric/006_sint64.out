sint64"0"
sint64"1"
sint64"12"
sint64"9223372036854775807"
sint64"-9223372036854775808"
sint64"0"
sint64"15"
sint64"15"
sint64"13"
sint64"2"
sint64"-123"
sint64"0"
sint64"10"
sint64"-10"
sint64"12"
sint64"6"
sint64"0"
sint64"-20"
sint64"20"
sint64"5"
sint64"4"
sint64"-5"
sint64"3"
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
64
true
